tests/integration/data/on-and-to.on-arm64-to-arm64.yaml

```diff
@@ -0,0 +1,5 @@
+plugin: nil
+source-branch: branch3
+build-packages:
+  - package9
+organize:
```

Comment:
I'm surprised by this! I'd have expected it to be `organize: {}`

Reply:
Agreed, this is also unexpected.  I'll make a bug for this too.